components/sysutils/virtualbox/Makefile

```diff
@@ -61,29 +88,58 @@ LOCAL_CONFIG+='\nVBOX_WITH_DTRACE_R0DRV ='
 LOCAL_CONFIG+='\nVBOX_WITH_DTRACE_RC ='
 LOCAL_CONFIG+='\nVBOX_WITH_NATIVE_DTRACE ='
 
-COMPONENT_POST_UNPACK_ACTION = ( cp $(COMPONENT_DIR)/files/Makefile $(COMPONENT_SRC))
-
 COMPONENT_PRE_CONFIGURE_ACTION= \
   ( cd $(@D); echo $(LOCAL_CONFIG) > LocalConfig.kmk; $(CP) -a $(SOURCE_DIR)/* $(@D) )
 
-
 CONFIGURE_OPTIONS = --with-gcc="$(CC)"
 CONFIGURE_OPTIONS+= --with-g++="$(CXX)"
-CONFIGURE_OPTIONS+= --build-headless
+CONFIGURE_OPTIONS+= --enable-qt5
 CONFIGURE_OPTIONS+= --disable-alsa
 CONFIGURE_OPTIONS+= --disable-dbus
 CONFIGURE_OPTIONS+= --disable-docs
-CONFIGURE_OPTIONS+= --disable-pulse
+CONFIGURE_OPTIONS+= --enable-pulse
 CONFIGURE_OPTIONS+= --disable-python
 CONFIGURE_OPTIONS+= --disable-sdl-ttf
 CONFIGURE_OPTIONS+= --disable-libvpx
 CONFIGURE_OPTIONS+= --nofatal
 
+VBOX_PREFIX   = /opt/VirtualBox
```

Comment:
Do we want to stick to /opt/VirtualBox layout or move it to /usr/VirtualBox to be more consistent with other packages?

Reply:
If we want another prefix we need to patch configuration and install scripts.
That would be better probably but I will not have time to look at this in the coming two weeks.